static/app/components/events/eventEntry.tsx

```diff
@@ -80,11 +84,16 @@ function EventEntryContent({
     case EntryType.EXPECTCT:
     case EntryType.EXPECTSTAPLE:
       const {data, type} = entry;
-      return <Generic type={type} data={data} />;
+      return <Generic sectionKey={FoldSectionKey.EXPECTCT} type={type} data={data} />;
```

Comment:
do you know if `EXPECTSTAPLE` is a case we need to cover here

Reply:
Missed it, good catch!